debbuild.spec

```diff
@@ -53,29 +51,10 @@ rebuild .src.rpm source packages as .deb binary packages.
 # Transfer $VERSION into the live system
 %{__perl} -pe "s/\@VERSION\@/%{version}/" -i debbuild
 
-%install
-# Steps to install to a temporary location for packaging
-%{__rm} -rf %{buildroot}
-
-%{__install} -d %{buildroot}%{_bindir}
-%{__install} debbuild %{buildroot}%{_bindir}
-
-%{__install} -d %{buildroot}%{_libdir}/%{name}
-%{__install} -m 644 macros/macros.in %{buildroot}%{_libdir}/%{name}/macros
-%{__install} -m 644 config/debrc %{buildroot}%{_libdir}/%{name}
+%configure NAME=%{name} VERSION=%{version}
 
-%{__install} -d %{buildroot}%{_sysconfdir}/%{name}
-%{__install} -m 644 macros/macros.sysutils %{buildroot}%{_sysconfdir}/%{name}
-%{__install} -m 644 macros/macros.texlive %{buildroot}%{_sysconfdir}/%{name}
-%{__install} -m 644 macros/platform.in %{buildroot}%{_sysconfdir}/%{name}/macros
-
-%{__install} -d %{buildroot}%{_mandir}/man8
-%{__pod2man} --utf8 --center="DeepNet Dev Tools" --section 8 \
-	--release="Release %{version}" debbuild \
-	%{buildroot}%{_mandir}/man8/debbuild.8
-
-%{__install} -d %{buildroot}%{_datadir}/locale/de/LC_MESSAGES
-%{__msgfmt} po/de/debbuild.po -o %{buildroot}%{_datadir}/locale/de/LC_MESSAGES/debbuild.mo
+%install
+make install DESTDIR=%{buildroot}
```

Comment:
`%make_install` should pass DESTDIR, right? If so I'll replace this with that.

Reply:
Yes, see line 306 of `macros.in`.

Reply:
Awesome, thanks. Replaced.